scripts/tokenomics-emulation/config.json

```diff
@@ -1,10 +1,14 @@
 {
-  "network": {},
+  "network": {
+    "url": "wss://rpc.testnet.cere.network:9945"
+  },
```

Comment:
Let's add `accounts` object at the same level as `network` and add there `root_account_mnemonic`.